source/partials/_blog-social-share.haml

```diff
@@ -0,0 +1,23 @@
+- post = current_page
+
+.blog-post__share-links
+  %a.blog-post__share-link.blog-post__share-link--facebook{ href: facebook_share_href(post), rel: 'noopener noreferrer', target: '_blank' }
+    = partial "images/icons/social/facebook.svg"
+
+  %a.blog-post__share-link.blog-post__share-link--twitter{ href: twitter_share_href(post), rel: 'noopener noreferrer', target: '_blank' }
+    = partial "images/icons/social/twitter.svg"
+
+  %a.blog-post__share-link.blog-post__share-link--linkedin{ href: linkedin_share_href(post), rel: 'noopener noreferrer', target: '_blank' }
+    = partial "images/icons/social/linkedin.svg"
+
+  %a.blog-post__share-link.blog-post__share-link--hackernews{ |
+      "data-url" => post.data.hn_url ? post.data.hn_url : '', |
+      "data-title" => post.data.title,                        |
+      href: hackernews_share_href(post),                      |
+      rel: 'noopener noreferrer',                             |
+      target: '_blank'                                        |
+    }                                                         |
```

Comment:
This is where that `hn_url` field can be used to have the hackernews submission link helper check a different URL for an existing submission.